src/screens/WalletConnect/CallRequest/WalletConnectCallRequestModal.js

```diff
@@ -52,13 +55,21 @@ type Props = {|
 function WalletConnectCallRequestModal({ request }: Props) {
   const { t } = useTranslation();
   const navigation = useNavigation();
+  const chainsConfig = useChainsConfig();
 
   const ref = React.useRef();
 
   const { rejectCallRequest } = useWalletConnect();
 
   const type = getWalletConnectCallRequestType(request);
-  const title = formatRequestType(type);
+  const chain = chainFromChainId[request.chainId] ?? CHAIN.ETHEREUM;
```

Comment:
I'm not sure if we should default to mainnet if request.chainId would indicate some other, unsupported chain. @IAmKio please advise.

Reply:
Replied to @divyaBacancy directly on this but we shouldn't switch the users chain without their knowledge programatically.